### ISSUE TITLE
Clicking back from a project to my projects takes an extra click in Firefox

### ISSUE THREAD

**Issue by jgmac1106** [mozilla/thimble.mozilla.org#1239]
![thimblebug](https://cloud.githubusercontent.com/assets/10534566/11021190/b1c6b3d0-8631-11e5-94f1-fd5115cd92c2.gif)

It does not happen in Chrome. 

Also I am wondering if the my project list should open in a blank target. I know we have said blank targets may violate best practice for accessibility but when sharing a lot of projects it was a PITA.

**Comment by gideonthomas** [comment]
I wonder if this is: http://stackoverflow.com/questions/5259154/firefox-back-button-vs-iframes

**Comment by humphd** [comment]
Is this still an issue?  Can someone try it on staging: https://bramble.mofostaging.net

**Comment by Pomax** [comment]
I cannot reproduce this anymore
